docs/risk/risk-faq.md

```diff
@@ -16,13 +16,13 @@ Yes, EigenLayer does reuse the same economic security by enabling different AVSs
 
 Pooling allows various AVSs to contribute to a larger, shared economic security base, enhancing economic safety. This is because the cost to compromise any individual AVS increases significantly. For instance, consider 100 AVSs each with \$1B in economic security versus 100 AVSs sharing \$100B in economic security. The cost to attack any single AVS in the pooled system has effectively multiplied by 100.
```

Comment:
This paragraph (and the one above) should be future tense
.... and should probably not talk about pooled security in such concrete terms

Reply:
I see this as "whitepaper kind of language"
Is it perhaps a bit more present-tense than it should be? Probably, but in my mind it is talking about the "idea" of EigenLayer.
I'm cool talking about this more, but would like to get the other changes merged aside from this discussion.